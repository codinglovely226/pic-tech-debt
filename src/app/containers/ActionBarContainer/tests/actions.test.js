import {
  toggleNewCallModal,
  updateNewCallModalCallerLocation,
  updateNewCallModalCallerOrganisation,
  updateNewCallModalCentreName,
  updateNewCallModalFilter,
  updateNewCallModalId,
  updateNewCallModalName,
  updateNewCallModalPostcode,
  updateNewCallModalSelected,
  updateNewCallModalSex,
  updateNewCallModalTime,
} from '../actions';

describe('ActionBarContainer action creators', () => {
  it('toggleNewCallModal', () => {
    expect(toggleNewCallModal()).toMatchSnapshot();
  });
  it('updateNewCallModalCallerLocation', () => {
    expect(updateNewCallModalCallerLocation('New Location')).toMatchSnapshot();
  });
  it('updateNewCallModalCallerOrganisation', () => {
    expect(updateNewCallModalCallerOrganisation('New organisation')).toMatchSnapshot();
  });
  it('updateNewCallModalCentreName', () => {
    expect(updateNewCallModalCentreName('New centre name')).toMatchSnapshot();
  });
  it('updateNewCallModalFilter', () => {
    expect(updateNewCallModalFilter('New filter')).toMatchSnapshot();
  });
  it('updateNewCallModalId', () => {
    expect(updateNewCallModalId('New id')).toMatchSnapshot();
  });
  it('updateNewCallModalName', () => {
    expect(updateNewCallModalName('New name')).toMatchSnapshot();
  });
  it('updateNewCallModalPostcode', () => {
    expect(updateNewCallModalPostcode(8000)).toMatchSnapshot();
  });
  it('updateNewCallModalSelected', () => {
    expect(updateNewCallModalSelected('New selected')).toMatchSnapshot();
  });
  it('updateNewCallModalSex', () => {
    expect(updateNewCallModalSex('New sex')).toMatchSnapshot();
  });
  it('updateNewCallModalTime', () => {
    expect(updateNewCallModalTime('New time')).toMatchSnapshot();
  });
});
