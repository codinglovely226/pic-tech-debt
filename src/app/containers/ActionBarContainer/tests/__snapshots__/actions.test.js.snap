exports[`ActionBarContainer action creators toggleNewCallModal 1`] = `
Object {
  "type": "pic/ActionBarContainer/TOGGLE_NEW_CALL_MODAL",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalCallerLocation 1`] = `
Object {
  "callerLocation": "New Location",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_CALLER_LOCATION",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalCallerOrganisation 1`] = `
Object {
  "callerOrganisation": "New organisation",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_CALLER_ORGANISATION",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalCentreName 1`] = `
Object {
  "centreName": "New centre name",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_CENTRE_NAME",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalFilter 1`] = `
Object {
  "filter": "New filter",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_FILTER",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalId 1`] = `
Object {
  "naturalId": "New id",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_ID",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalName 1`] = `
Object {
  "name": "New name",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_NAME",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalPostcode 1`] = `
Object {
  "postcode": 8000,
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_POSTCODE",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalSelected 1`] = `
Object {
  "selected": "New selected",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_SELECTED",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalSex 1`] = `
Object {
  "sex": "New sex",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_SEX",
}
`;

exports[`ActionBarContainer action creators updateNewCallModalTime 1`] = `
Object {
  "time": "New time",
  "type": "pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_TIME",
}
`;
